tox.ini

```diff
@@ -4,17 +4,12 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py26,py27,cover,lint
+envlist = py26,py27,lint
```

Comment:
I was afraid that "it is just showing coverage" vs. failing if coverage is below the required threshold (100% in our case, although temporarily lowered by @jdkasten to have more green in Travis CI :)) and thus by including this commit we will no longer track (through Travis) coverage properly: users will be able to submit PRs without tests that will show up as green :(